PS-1769: Validation Endpoint (#44)
connective-health/public-docs: api-reference/release-notes.mdx

icon: 'gears'
---

<Update label="2025-03-14" description="v3.1.0">

    ## New Features

>     * New [Patient Data v2 Validate](/clinical-data/patient-data-v2-validate) endpoint which may be used to check the validity of one or more processing records without attempting to actually process them

Reviewer: ```suggestion
    * New [Patient Data v2 Validate](/clinical-data/patient-data-v2-validate) endpoint that checks the validity of one or more processing records without actually processing them
```
(records or requests?)
